examples/beerus-rpc/starknet_getBlockTransactionCount.hurl

```diff
@@ -2,6 +2,6 @@ POST http://0.0.0.0:3030
 Content-Type: application/json
 {
     "jsonrpc":"2.0","method":"starknet_getBlockTransactionCount","params":[
-        "tag", "latest"
+        "latest"
```

Comment:
Valid also for all other instances: Here you do not follow the same structure as in the other hurls for example: 
{
      "block_number": 22050
},

After testing it does work but is that normal ? 

Shouldn't it be: 

{
 "block_tag": "latest"
}

Or something in that range ?

Reply:
<img width="620" alt="Screen Shot 2023-04-13 at 2 42 44 PM" src="https://user-images.githubusercontent.com/14187644/231889373-3f1855f9-ab2f-45a9-a2fb-18e4ea3b364b.png">

That's the way it's done in the ecosystem